README.md

```diff
@@ -92,6 +111,9 @@ brew install slither-analyzer
 
 ```bash
 git clone https://github.com/crytic/slither.git && cd slither
+# Using uv (recommended)
+uv pip install .
+# Or using pip
 python3 -m pip install .
```

Comment:
this should probably be installed as editable so when you do development, your changes actually get reflected.

the uv way of doing this would be to use the `uv tool install` command
```console
uv tool install .
```
this installs the slither commands for your current user as an "editable" package which is convenient for testing/development

or using uv run without installing it to your current user
```console
uv run <slither-command>
```
uv run with make the venv in the current directory under .venv and automatically sync everything and install the editable version of the package